src/styles/blocks/product-card.css

```diff
@@ -0,0 +1,117 @@
+.product-card {
+  width: 200px;
+  height: 408px;
+
+  position: relative;
+  display: flex;
+  flex-direction: column;
+
+  font-family: Roboto, Arial, sans-serif;
+  color: #060b35;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.product-card__image-container {
+  display: flex;
+  justify-content: center;
+}
+
+.product-card__image {
+  width: 160px;
+  height: 134px;
+
+  margin: 16px auto 40px;
+}
+
+.product-card__info {
+  display: flex;
+  flex-direction: column;
+
+  margin: 0 0 16px;
+}
+
+.product-card__name-link {
+  font-weight: 500;
+  font-size: 12px;
+  color: #060b35;
+  text-decoration: none;
+
+  line-height: 150%;
+
+  margin: 0 0 4px;
+}
+
+.product-card__code {
+  font-weight: 400;
+  font-size: 10px;
+
+  color: #616070;
+}
+
+.product-card__rate {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+
+  margin: 0 0 24px;
+}
+
+.product-card__reviews-link {
+  font-weight: 400;
+  font-size: 10px;
+  color: #060b35;
+
+  text-decoration: none;
+}
+
+.product-card__price {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+
+  font-weight: 700;
+  font-size: 16px;
+
+  margin: 0 0 16px;
+}
+
+.product-card__price > :first-child {
+  font-weight: 400;
+  font-size: 12px;
+  color: #616070;
+}
+
+.product-card__buy-button {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  width: 100%;
+  height: 40px;
+
+  background: #00acdc;
+  border: 1px solid #00acdc;
+  border-radius: 5px;
+
+  font-weight: 700;
+  font-size: 14px;
+  color: #fff;
+  text-decoration: none;
+  text-transform: uppercase;
+
+  transition: 1s;
+}
+
+.product-card__buy-button:hover {
+  color: #00acdc;
+
+  background: #fff;
+  border: 1px solid #00acdc;
```

Comment:
change only border color, you already set it line 97.

According to checklist point 9: [STYLES] - Don't add new border to the element on hover. Add default transparent border of the same width, and change its color on :hover